fish_prompt.fish

```diff
@@ -92,8 +92,19 @@ function __bobthefish_pretty_parent -S -a current_dir -d 'Print a parent directo
   string replace -ar '(\.?[^/]{'"$fish_prompt_pwd_dir_length"'})[^/]*/' '$1/' "$parent_dir/"
 end
 
+function __ignore_vcs_dir -d 'Identifies if dir is to be ignored for VCS highlighting'
+  for path_ in $vcs_ignore_paths
```

Comment:
we try to use a `theme_` prefix for as many theme related config options as possible. should this use that prefix as well?

Reply:
I suspected as much but wasnt sure, I will rename it..